test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -24,17 +23,13 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
 
         public HttpClient Client { get; }
 
-#if NETCOREAPP1_0
         [Theory]
-#else
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "aspnet/Mvc#3587")]
```

Comment:
Entirely possible. Wasn't a major concern and I certainly didn't try to confirm problem remained recently.